node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/configserver/noderepository/Acl.java

```diff
@@ -32,34 +33,39 @@ public Acl(List<Integer> trustedPorts, List<InetAddress> trustedNodes) {
         this.trustedPorts = trustedPorts != null ? ImmutableList.copyOf(trustedPorts) : Collections.emptyList();
     }
 
-    public String toRules(IPVersion ipVersion) {
+    public List<String> toRules(IPVersion ipVersion) {
+        List<String> rules = new LinkedList<>();
```

Comment:
(Why LinkedList?)

Reply:
Because the final list size depends on how many IPs we have, and we only use this list to add elements which works well with LinkedList as we don't need to resize underlying array. 

Although I see now that calculating final list size is easy, as it is some constant + number of trusted nodes, so could use ArrayList with predefined size.